dwarf/dwarf.cc

```diff
@@ -273,12 +273,23 @@ compilation_unit::compilation_unit(const dwarf &file, section_offset offset)
         cursor sub(subsec);
         sub.skip_initial_length();
         uhalf version = sub.fixed<uhalf>();
-        if (version < 2 || version > 4)
-                throw format_error("unknown compilation unit version " + std::to_string(version));
+        (void)version;
+        /*if (version < 2 || version > 4)
```

Comment:
I'd rather this continue to throw if version > 5, rather than fail in far less obvious ways if they change the header again.

Reply:
Fine with me